src/frontend/contexts/dataContext.js

```diff
@@ -7,6 +7,7 @@ const DataContext = createContext();
 
 const DataProvider = ({ children }) => {
     let [ videos , setVideos] = useState([]);
+    let [ selectedVideo , setSelectedVideo ] = useState('none');
```

Comment:
You can use const here as well, le doen't seem to be required here

Reply:
hi Om, surely will look into it and change it